templates/404.html

```diff
@@ -3,9 +3,9 @@
 <html lang={{ config.lang_code }} dir={{ lang_dir }}>
   <head>
     {% block head_extras %}{% endblock %}
-    <title>Citation Hunt</title>
+    <title>{{ config.strings.tooltitle }}</title>
     <meta name="robots" content="nofollow, noodp, noindex">
-    <meta name="description" content="Citation Hunt - {{ config.strings.introduction }}">
+    <meta name="description" content="{{ config.strings.tooltitle }} - {{ config.strings.introduction }}">
```

Comment:
I'd be fine with either just removing the `{{ config.strings.tooltitle }}` part, or changing the `introduction` string to contain a `%s`, then interpolate.

`introduction` is actually deprecated in the UI; most of the languages use `instructions_goal` and `instructions_detail` in its place, and I forgot we even used it in the `meta` tag.

Even if someone were to update `introduction` in one of the languages that lag behind, they'd probably do the others at the same time anyway, and it would disappear from the UI, so I don't worry much about changing it.